WordPress/Tests/XSS/EscapeOutputUnitTest.inc

```diff
@@ -60,4 +60,12 @@ echo $foo ? $foo : 'no foo'; // Bad
 echo empty( $foo ) ? 'no foo' : $foo; // Bad
 echo $foo ? esc_html( $foo ) : 'no foo'; // OK
 
-echo 4; // OK
\ No newline at end of file
+echo 4; // OK
+
+exit( $foo ); // Bad
+exit( esc_html( $foo ) ); // OK
+
+die( $foo ); // Bad
+die( esc_html( $foo ) ); // OK
+
+printf( 'Hello %s', $foo ); // Bad
```

Comment:
I've just pushed some improved tests in 39881e8 that demonstrate that this works for `printf()`. It also addresses `vprintf()`. `sprintf()` is still another story, which I/someone will tackle at a later date. :-)